components/l3d/src/L3DFile.cpp

```diff
@@ -473,6 +473,46 @@ void L3DFile::ReadFile(std::istream& stream)
 		}
 	}
 
+	// Get additional data. Strictly in this order
+	// Footprint data
+	uint32_t additionalDataOffset = 0;
+	uint32_t footprintDataSize = 0;
+	if (static_cast<uint32_t>(_header.flags) & static_cast<uint32_t>(L3DMeshFlags::ContainsLandscapeFeature))
+	{
+		stream.seekg(0x48, stream.beg);
```

Comment:
Where does this magic number come from?
Could you make a const with its derivation available? Either a bunch of sizeofs or offsetofs would make it easier to understand for others.